presto-spi/src/main/java/com/facebook/presto/spi/function/Parameter.java

```diff
@@ -21,13 +21,13 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class SqlParameter
+public class Parameter
```

Comment:
I understand. The reason I'm renaming here is that I'm reserving the name `SqlParameter` for the new annotation. We have annotations like `@SqlType`, `@SqlNullable`, `@SqlFunction` and so on, to define functions. Here we need `@SqlParameter` and `@SqlParameters`, and hence the rename.

Reply:
I understand the motivation. And for this specific case I think it's probably ok to break spi since I don't expect people to use it yet. But in general, having naming conventions consistent and nice is not enough reason to break backward compatibility.